static/js/components/ClassificationForm.jsx

```diff
@@ -0,0 +1,199 @@
+import React, { useReducer } from 'react';
+import { useDispatch } from 'react-redux';
+import PropTypes from 'prop-types';
+import Select from "@material-ui/core/Select";
+import InputLabel from "@material-ui/core/InputLabel";
+import TextField from '@material-ui/core/TextField';
+import Button from '@material-ui/core/Button';
+import MenuItem from "@material-ui/core/MenuItem";
+import Autocomplete from '@material-ui/lab/Autocomplete';
+import { showNotification } from "baselayer/components/Notifications";
+import * as Actions from '../ducks/source';
+
+
+function makeMenuItem(taxonomy, index) {
```

Comment:
This should either be refactored into a react component, or, preferably, just move the JSX down to the main return statement because it's so small.

Reply:
done.